src/svg/SkSVGDevice.cpp

```diff
@@ -655,6 +662,48 @@ SkString SkSVGDevice::AutoElement::addLinearGradientDef(const SkShaderBase::Grad
     return id;
 }
 
+SkString SkSVGDevice::AutoElement::addRadialGradientDef(const SkShaderBase::GradientInfo& info,
+                                                        const SkShader* shader,
+                                                        const SkMatrix& localMatrix) {
+    SkASSERT(fResourceBucket);
+    SkString id = fResourceBucket->addLinearGradient();
+
+    {
+        AutoElement gradient("radialGradient", fWriter);
+
+        gradient.addAttribute("id", id);
+        gradient.addAttribute("gradientUnits", "userSpaceOnUse");
+        gradient.addAttribute("cx", info.fPoint[0].x());
```

Comment:
at Rive we don't support an end circle position, so cx and cy need to point to the start circle as well.